api/openapi.yaml

```diff
@@ -1056,10 +1056,18 @@ components:
         endTime:
           type: string
           format: date-time
+        timeoutTime:
+          description: |
+            A time in the future when this Scan will be automatically aborted
+            if it is still in progress.
+          type: string
+          format: date-time
```

Comment:
I'm torn whether to make this a timeoutTime or timeoutSeconds. From a API UX perspective timeoutSeconds is better because you don't need to calculate a time before posting it to the API, and then we can compare those seconds to the startTime to know if we've reached timeout.

I think timeoutSeconds will be better when we have to ability to POST a Scan directly without a ScanConfig.

Reply:
I'll remove it completely and keep the `timeoutSeconds` as part of the `ScanConfigSnapshot`. `Scan` objects must still have the `ScanConfigSnapshot` attribute even if the there is no parent `ScanConfig` for them.

Reply:
👍 sounds good. The re-model should make this a little less weird

Reply:
Fixed.